cdap-explore/src/main/java/co/cask/cdap/explore/service/ExploreTableManager.java

```diff
@@ -204,6 +204,13 @@ public QueryHandle enableDataset(Id.DatasetInstance datasetID, DatasetSpecificat
     }
   }
 
+  private boolean isExplorableTable(String datasetType) {
+    return ObjectMappedTableModule.FULL_NAME.equals(datasetType) ||
+      ObjectMappedTableModule.SHORT_NAME.equals(datasetType) ||
+      "table".equals(datasetType) ||
+      Table.class.getName().equals(datasetType);
```

Comment:
This PR enables more tables like `IndexedTable`, etc. to have schema so that they can be explored, right? Should we say any dataset that implements Table and has a schema is explorable?